lib/reek/context/class_context.rb

```diff
@@ -0,0 +1,11 @@
+require_relative 'module_context'
+
+module Reek
+  module Context
+    #
+    # A context wrapper for any class found in a syntax tree.
+    #
+    class ClassContext < ModuleContext
```

Comment:
If this is the case doesn't it make more sense to move `#method_calls` here?

(But perhaps this smell should also check modules?)